src/components/ActivityDetails/ActivityDetails.test.tsx

```
@@ -236,12 +242,25 @@ jest.mock('../CorrectActivityModal/CorrectActivityModal', () => ({
           onClick={() =>
             onObservationSubmit?.(data?.studentId || '', 'With file', [
```

ESLint

[error] Replace `data?.studentId·||·'',·'With·file',·[` with `⏎··············data?.studentId·||·'',⏎··············'With·file',`

(prettier/prettier)